Gruntfile.coffee

```diff
@@ -34,12 +34,11 @@ module.exports = (grunt) ->
             dest + matchedSrcPath
           )
       test:
-        files: grunt.file.expandMapping(['**/*.spec.coffee'], 'test/',
+        files: grunt.file.expandMapping(['**/*.coffee'], 'test/',
           flatten: false
           cwd: 'src/spec'
-          ext: '.spec.js'
           rename: (dest, matchedSrcPath) ->
-            dest + matchedSrcPath
+            dest + matchedSrcPath.replace('coffee', 'js')
```

Comment:
Instead of doing this replace and removing `ext` parameter totally, what if we add the ext with `.js` only

`ext: '.js'`